src/main/kotlin/template/MyAwesomeDokkaPlugin.kt

```diff
@@ -1,7 +1,11 @@
 package template
 
 import org.jetbrains.dokka.plugability.DokkaPlugin
+import org.jetbrains.dokka.plugability.DokkaPluginApiPreview
+import org.jetbrains.dokka.plugability.PluginApiPreviewAcknowledgement
 
 class MyAwesomeDokkaPlugin : DokkaPlugin() {
 
-}
\ No newline at end of file
+    @OptIn(DokkaPluginApiPreview::class)
+    override fun pluginApiPreviewAcknowledgement() = PluginApiPreviewAcknowledgement
```

Comment:
Not sure if we should opt-in by default in the template

Reply:
Decided not to opt in by default, and instead give the user the opportunity to do so themselves